### PR TITLE
GCP Launch Tweaks

### PR DESCRIPTION
**Pull request by josh-padnick**
This PR adds a number of updates that improve our readiness to support GCP customers:

- Merge #117 
- Merge #104
- Lots of clean up to pricing page
- Updated pricing FAQs
- Terms of Service updated to include support for GCP
- Replace all instances of Dedicated Support with Professional Support
- Update Professional Support  to exclude phone/video
- Update checkout screen to better support enterprise support
- Verified that ChargeBee works correctly

This has not yet been deployed live. Feedback welcome!

### Update
As requested, here are some screenshots of all the changes:

**Pricing/Before:**

Note that a bunch of tooltips were updated on the "after" page as well, which of course aren't visible in the screenshots.

gruntworkiopricing-full

**Pricing/After:**

f3da9b17ngrokiopricing-full

**Terms of Service/After:*

Terms of Service now has a CHANGELOG. This update was added by Akos, and I just cleaned it up:

image

image

### CODE REVIEW

File: assets/js/checkout.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ $(function () {
   switch ($.query.get('addon')) {
     case 'subscription-type': _updateCheckout({ subscription_type: true }); break;
     case 'setup-deployment': _updateCheckout({ setup_deployment: true }); break;
-    case 'dedicated-support': _updateCheckout({ dedicated_support: true }); break;
+    case 'pro-support': _updateCheckout({ pro_support: true }); break;
```
Comment (brikis98) [inline assets/js/checkout.js]: Did you remember to update the incoming "getting started" links with this change?

Comment (josh-padnick) [reply assets/js/checkout.js]: I did indeed!

**Comment by josh-padnick** [comment]
Ok, screen caps posted in the original PR message.

**Comment by robmorgan** [comment]
Thanks for the screen caps, makes it much easier to digest.

File: assets/css/global.less [hunk 2]
```diff
@@ -1963,22 +1963,16 @@ input[type="radio"] .styled:checked + label::after {
...
 
 //// Pricing page
-.houston-features {
-  height: 255px;
-}
-
-.plans-heading {
-  //height: 170px;
-}
-
```
Comment (robmorgan) [inline assets/css/global.less]: Thanks for looking into this!!

File: assets/css/pages/pricing.less [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  // Center the "New" badge just above the text and below the GCP logo
+  #gcp-new {
+    position: absolute;
+    left: 50%;
+    top: -12px;
+    transform: translateX(-50%);
+  }
+
+  // Center the "Special Pricing" badge just above the GCp price
```
Comment (robmorgan) [inline assets/css/pages/pricing.less]:
```suggestion
  // Center the "Special Pricing" badge just above the GCP price
```

**Comment by josh-padnick** [comment]
Everyone's had an opportunity to respond and all comments addressed, so merging and deploying now!

**Pull request merged**